test/cctest/node_test_fixture.h

```diff
@@ -89,7 +73,8 @@ class NodeTestFixture : public ::testing::Test {
     platform_ = new node::NodePlatform(8, nullptr);
     v8::V8::InitializePlatform(platform_);
     v8::V8::Initialize();
-    params_.array_buffer_allocator = &allocator_;
+    params_.array_buffer_allocator =
+        v8::ArrayBuffer::Allocator::NewDefaultAllocator();
```

Comment:
I’m not sure it matters, but should we free the allocator instance at some point?

Reply:
Good point, I'll take a look and see. Thanks